server/database/models/schedule.ts

```diff
@@ -9,10 +9,14 @@ const schedule = new Schema({
     type: String,
     required: false,
   },
+  questionCategory: {
+    type: String,
+    required: false,
+  },
```

Comment:
Add options for `languages`, `specialization`, and `questionCategory` in this model.